tests/release/pipelines/push_to_external_registry.go

```diff
@@ -142,12 +142,12 @@ var _ = framework.ReleasePipelinesSuiteDescribe("[HACBS-1571]test-release-e2e-pu
 			"mapping": map[string]interface{}{
 				"components": []map[string]interface{}{
 					{
-						"component":  compName,
-						"repository": releasedImagePushRepo,
+						"name":  compName,
+						"repository": "quay.io/" + utils.GetEnv("DEFAULT_QUAY_ORG", "redhat-appstudio-qe")+ "/dcmetromap",
 					},
 					{
-						"component":  additionalComponentName,
-						"repository": additionalReleasedImagePushRepo,
+						"name":  additionalComponentName,
+						"repository": "quay.io/" + utils.GetEnv("DEFAULT_QUAY_ORG", "redhat-appstudio-qe")+ "/dimplepython",
```

Comment:
Any reason we need to use the env variable instead of just having a constant for these? Just wondering about future tests that will also need these values

Reply:
The original url is quay.io/redhat-appstudio-qe/dcmetromap, I don't have permission to write to the repo.  And because the secret "hacbs-release-tests-token" being used in the script is created from QUAY_TOKEN which we always set it to the QE's personal quay account. So, here we need to set the repo from QE's personal account to have write permission.    So, I updated to "quay.io/" + utils.GetQuayIOOrganization()+ "/simplepython".  utils.  Thanks

Reply:
Considering more, if we still use "quay.io/redhat-appstudio-qe/dcmetromap" as the repo, we must set QUAY_TOKEN to the one from redhat-appstudio-qe account to run this test case (we usually set the QUAY_TOKEN to our personal one in other tests).  It  can work also.  I  can change it back. Wdyt? @johnbieren

Reply:
I will approve for now and if we decide to change it we can, but it would be nice to have this merged so the test passes

Reply:
Then we can go with this fix. Thanks